tests/l2/qat/qatlib_build.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: intel-qat-workload
+  namespace: intel-qat
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: intel-qat-workload
+  namespace: intel-qat
+spec:
+  triggers:
+    - type: "ConfigChange"
+    - type: "ImageChange"
+  runPolicy: "Serial"
+  source:
+    type: Dockerfile
+    dockerfile: |
+        
+        FROM registry.access.redhat.com/ubi8/ubi AS builder
+        ARG QATLIB_VERSION="23.02.0"
+
+        RUN dnf -y update && \
+            dnf install -y gcc \
+              systemd-devel \
+              make \
+              automake \ 
+              autoconf \
+              libtool \
+              openssl-devel \
+              http://mirror.centos.org/centos/8-stream/PowerTools/x86_64/os/Packages/nasm-2.15.03-3.el8.x86_64.rpm \
```

Comment:
for the purpose of this build, I'd think we should go without `nasm`. See https://github.com/intel/qatlib/blob/39e19d4fc09345bb857ee8745619fa0f83f6f824/INSTALL#L224-L225

Reply:
got it, thanks. will update

Reply:
> for the purpose of this build, I'd think we should go without `nasm`. See https://github.com/intel/qatlib/blob/39e19d4fc09345bb857ee8745619fa0f83f6f824/INSTALL#L224-L225

Looks like the option is for the situation nasm compiler is not present in the build environment. If we have the nasm compiler, we should install the nasm and build the lib with fast-crc-in-assembler, otherwise, there might be some performance penalty.

Reply:
For this "test application", it's acceptable but pulling rpms from random urls is not.

Reply:
Can look for official links of nasm. nasm & qatlib is on codeready-builder-for-rhel-8-x86_64-rpms repo for ubi images. With a dockerfile this repo can be accessed and built on RHEL systems enabled with subscription manager. Via buildconfigs, subscription manager and its supported repos can be enabled only via RH account personal keys. Plan to enable for future releases.

Reply:
> For this "test application", it's acceptable but pulling rpms from random urls is not.

I agree. For the qatlib, the user should use RH distributed qatblib rpm package and shoul not build the lib by themselves. However, currently, it is not very easily for user to do that.The [enhancement](https://github.com/intel/intel-technology-enabling-for-openshift/issues/139) is filed for Milestone 1.1.0. to address the gap.

All in all, for the testing of the first QAT release(1.0.1), this test case is good enough. 

@mythi Thank you very much for your comments and help.